frontend/src/app/main/ui/workspace/tokens/changes.cljs

```diff
@@ -173,22 +174,30 @@
                        (zipmap (repeat value)))]
     {:layout-gap layout-gap}))
 
+(defn shape-ids-with-layout [state shape-ids]
```

Comment:
Should be marked as private as this function is only used here. It helps detect unused functions when they become unused thanks to linter.